lib/manager/cocoapods/__snapshots__/artifacts.spec.ts.snap

```diff
@@ -15,12 +15,6 @@ exports[`.updateArtifacts() catches write error 2`] = `Array []`;
 
 exports[`.updateArtifacts() dynamically selects Docker image tag 1`] = `
 Array [
-  Object {
-    "cmd": "docker ps --filter label=renovate_child -aq",
-    "options": Object {
-      "encoding": "utf-8",
-    },
-  },
```

Comment:
`removeDanglingContainer` is called after `setAdminConfig` only in `renovateRepository()` call